powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1400,6 +1400,218 @@ function UpdatePolicyEnvironmentsForTeams
     Write-Host "UpdatePolicyEnvironmentsForTeams completes."
 }
 
+function EnableManagementForEnvironment
```

Comment:
I know it will sound weird, but the feature name is "Managed Environments". So I suggest to change the name of the function to "EnableManagedEnvironmentsOnEnvironment". "EnableManagedEnvironments" also works.